script/openqa-bootstrap

```diff
@@ -9,7 +9,7 @@ source /etc/os-release
 if [[ "$NAME" == "openSUSE Leap" ]] ; then
 	zypper -n addrepo -p 90 obs://devel:openQA devel:openQA
 	zypper -n addrepo -p 91 obs://devel:openQA:Leap:${VERSION} devel:openQA:Leap:${VERSION}
-	zypper -n  --gpg-auto-import-keys refresh
+	zypper -n --gpg-auto-import-keys refresh
```

Comment:
unrelated change

Reply:
yes it is but also trivial.i dont think it 's worth separate ticket. i have restored it for now

Reply:
Sure, I only meant a separate *commit*, you can still have it in this PR of course